src/library_syscall.js

```diff
@@ -38,7 +38,10 @@ var SyscallsLibrary = {
         }
         return dir;
       }
-      return PATH.join2(dir, path);
+      if (!dir.endsWith("/")) {
+        dir += "/"
+      }
+      return dir + path;
```

Comment:
How is this different to the old code?

Reply:
That'll probably work fine, I put it in and will let the CI say.